core/src/main/scala/vault/op.scala

```diff
@@ -130,6 +130,15 @@ object Vault {
     roleName: String
   ): VaultF[Unit] = Free.liftF(DeleteKubernetesRole(authClusterName, roleName))
 
+  // TODO(drewgonzales360): Do some sort of validation to ensure the roleName is a unitName
```

Comment:
Remove this line, as there's no validation here - the vault subsystem is a generic vault client.